dev-libs/starpu/starpu-1.2.6-r1.ebuild

```diff
@@ -81,9 +81,6 @@ src_test() {
 src_install() {
 	default
 	use doc && dodoc -r doc/doxygen/*.pdf doc/doxygen/html
-	if use examples; then
-		insinto /usr/share/doc/${PF}/examples
-		doins -r examples/*
-	fi
+	use examples && dodoc -r examples && docompress -x /usr/share/doc/${PF}/examples
```

Comment:
Please don't chain `&&` like that. `if` is much more readable.

Reply:
Okay, fine for me. Seems, it's a personal prefence, as some people tend to ask me to use more ```&&``` and some more ```if```` ;-) Done.